### PR TITLE
develop

### PR DESCRIPTION
**Pull request by Layrovell**
[DEMO](https://Layrovell.github.io/react_dynamic-list-of-goods/)

### CODE REVIEW

**Review by xipholena — APPROVED** [review]
Well done!

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,51 @@
...
-// or
-// import * as goodsAPI from './api/goods';
+  getRedHandler = () => {
+    getAll()
+      .then((goods) => {
+        this.setState({
+          goods: goods.filter(good => good.color === 'red'),
+        });
+      });
+  };
```
[the remark this answers is not in the document]
Comment (aliceblunt8) [reply src/App.js:36]: filter/sort goods in api file

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,46 @@
...
+  render() {
+    return (
+      <div>
+        <h1>Dynamic list of Goods</h1>
+        <GoodList goods={this.state.goods} />
+        <button
+          onClick={() => this.selectGoods(getAll)}
+          type="button"
+        >
+          getAll
```
Comment (varseniuk) [inline src/App.js:27]:
```suggestion
          Get all
```
It's just text on the button. You can name it whatever you like. No need for camelCase

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,46 @@
...
+        </button>
+        <button
+          onClick={() => this.selectGoods(get5FirstHandler)}
+          type="button"
+        >
+          get5First
```
Comment (varseniuk) [inline src/App.js:33]:
```suggestion
          Get 5 first
```

File: src/App.js [hunk 4]
```diff
@@ -1,13 +1,46 @@
...
+        </button>
+        <button
+          onClick={() => this.selectGoods(getRedHandler)}
+          type="button"
+        >
+          getRed
```
Comment (varseniuk) [inline src/App.js:39]:
```suggestion
          Get only red
```

**Review by varseniuk — APPROVED** [review]
👍  Good job.
Read small comment about naming

**Pull request closed without merging**